src/common/error_reporting.rs

```diff
@@ -0,0 +1,28 @@
+pub struct ErrorReporter;
+
+impl ErrorReporter {
+    fn get_url() -> String {
+        if cfg!(debug_assertions) {
+            "https://staging-telemetry.qdrant.io".to_string()
+        } else {
+            "https://telemetry.qdrant.io".to_string()
+        }
+    }
+
+    pub fn report(error: &str, reporting_id: &str, backtrace: Option<&str>) {
+        let client = reqwest::blocking::Client::new();
+
+        let report = serde_json::json!({
+            "id": reporting_id,
+            "error": error,
+            "backtrace": backtrace.unwrap_or(""),
+        });
+
+        let data = serde_json::to_string(&report).unwrap();
+        let _resp = client
```

Comment:
Should we maybe log failures?

Reply:
no, reporting in 100% optional, in some cases it is expected not to work (like no internet env, for example). So we won't log anything here